### PR TITLE
IAP-5084: add documentation on session hijack prevention

### PR DESCRIPTION
**Pull request by kvedenskii-axway**
https://deploy-preview-2427--axway-open-docs.netlify.app/docs/apim_installation/apiportal_install/secure_harden_portal/#enable-session-hijack-prevention

### CODE REVIEW

File: content/en/docs/apim_installation/apiportal_install/secure_harden_portal.md [hunk 1]
```diff
@@ -276,6 +276,13 @@ The following options are available from the **Authentication method** list:
...
 * Yubikey: Allows you to use a Yubikey secure hardware token for two-factor authentication. In addition to the username and password, it also requires you to insert your Yubikey into the USB port of your computer (Click the **Secret Key** area of the site's login area and touch the **Yubikey's gold disk**). If you have an NFC-equipped Android smartphone you can just hold a compatible Yubikey token (Yubikey Neo) close to the NFC reader to copy the secret code to the device's clipboard. The secret code generated by your Yubikey is unique to your device, and it is constantly changing. You can enable two-factor authentication for the front-end, back-end, or for both. You can configure this in the plugin **Two-Factor Authentication – Yubikey** plug-in.
 
+## Enable Session Hijack Prevention
+
+1. Log in to JAI.
+2. Select **Components > API Portal > Additional Settings**.
+3. Under **Session Hijack Prevention** section select **Yes** for **Enabled**.
```
Comment (dginevaAxway) [inline content/en/docs/apim_installation/apiportal_install/secure_harden_portal.md:283]: what is the default value, it should be enabled?

Comment (varlogerr) [reply content/en/docs/apim_installation/apiportal_install/secure_harden_portal.md:283]: Correct, it comes enabled by default.

Comment (kvedenskii-axway) [reply content/en/docs/apim_installation/apiportal_install/secure_harden_portal.md:283]: Correct, it comes enabled by default.

Comment (dginevaAxway) [reply content/en/docs/apim_installation/apiportal_install/secure_harden_portal.md:283]: @andreamussap  do you think is worth mentioning in the doc that the feature is enable by default while is obviously visible when you visit the API Portal settings.

Comment (andreamussap) [reply content/en/docs/apim_installation/apiportal_install/secure_harden_portal.md:283]: @dginevaAxway @kvedenskii-axway all sections need an introductory sentence, so please add some brief introduction.

I don't understand, if the feature is enabled by default, why do you have to select Yes for enabled ?

Comment (kvedenskii-axway) [reply content/en/docs/apim_installation/apiportal_install/secure_harden_portal.md]: @andreamussap updated the section

**Pull request merged**